### PR TITLE
add 2 sec with mobile styles

### PR DESCRIPTION
**Pull request by Mov305**
Added 2 sections
   1-Project section.
   2-About me section.
Added mobile styles
including 
- Grid CSS
- Flex box CSS

### CODE REVIEW

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Changes Requested :recycle:

almost there

## Hey @Mov305 :wave:
### Highlights :1st_place_medal:

Your project is working :clap:
Good commit history :heavy_check_mark:
Following Github Flow :heavy_check_mark:
Following HTML, CSS best practices :heavy_check_mark:
Using .gitignore file :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

You have made a great effort on your project :blush: and I can see that but you need to work on some issues to go to the next project but you are almost there. :+1:

### **_Any requested change with the optional tag is not crucial enough to prevent you from getting approval. But we, highly recommend that you take these changes into consideration_**

### Once you implement all the changes you should be ready to submit your project for a re-review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -170,10 +170,10 @@ <h5 class="pointer">LET'S CONNECT</h5>
               <img src="./assets/angellist.svg" alt="Angellist" />
             </li>
           </ul>
-          <h5 class="btn">Get my resume</h5>
+          <a class="btn">Get my resume</a>
         </div>
-        <section id="lfs-section">
-          <div id="languages">
+        <ul id="lfs-section">
+          <li id="languages">
```
Comment (isaacpitwa) [inline index.html:176]: Screenshot 2022-07-01 at 02 10 33

- [ ] As pointed out by the previous reviewer. unfortunately, this issue was not resolved with the latest changes.

File: index.html [hunk 2]
```diff
@@ -59,7 +59,7 @@ <h5 class="pointer">LET'S CONNECT</h5>
         </section>
       </main>
       <!--Project section-->
-      <div id="projects">
+      <section id="projects">
```
Comment (isaacpitwa) [inline index.html:62]: - [ ] As pointed out  by the previous reviewer [here](https://github.com/Mov305/P1-portofolio/pull/3#discussion_r911434381). Unfortunately, this issue was not solved by the latest changes. You could try adding extra padding to the projects section.

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi @Mov305 🙂 ,

Great work on making the changes requested by a previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
- Meaningful commit messages ✅

## Required changes ♻️ 

Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mov305) [reply index.html:176]: I already resolve it are you sure your opening the latest update?

**Review by codecaiine — APPROVED** [review]
Hi @Mov305 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello, @Mov305  👋,

_This was supposed to be an additional review as a result of the second opinion request._

Kindly, after you ask for a second opinion review, don't commit other changes to the milestone nor ask again for another review.
Please assume your decision and exercise patience.

Anyways, I am glad to see that your project got approved.

Keep rocking 🚀

**Pull request merged**